apps/sms/test/marionette/lib/views/inbox/view.js

```diff
@@ -11,28 +11,56 @@ function InboxView(client) {
 
 InboxView.prototype = {
   get conversations() {
-    return this.accessors.conversations.map(function(conversation) {
-      return {
-        lastMessageType: conversation.getAttribute('data-last-message-type'),
-        title: this.accessors.getConversationTitle(conversation).text()
-      };
-    }, this);
+    return this.accessors.conversations.reduce(function(list, conversation) {
+      // It's workaround for the case when conversation node reference is stale,
```

Comment:
Here's what I was fighting with and likely the reason of recent intermittent failures in drafts tests - when draft is updated/removed we remove entire node and create a new one - so there is chance that we get conversation node before it has been removed and will try tap on it (to navigate inside conversation for example) when it has been already removed.